src/components/ftbot/PairlistConfigurator.vue

```diff
@@ -66,19 +66,26 @@
         />
       </div>
     </div>
-    <div class="col-12 col-lg-3">
+    <div class="d-flex flex-column col-12 col-lg-3">
       <b-form-radio-group v-model="selectedView" class="mb-2" size="sm" buttons>
         <b-form-radio button value="Config"> Config</b-form-radio>
         <b-form-radio button value="Results" :disabled="pairlistStore.whitelist.length === 0">
           Results</b-form-radio
         >
       </b-form-radio-group>
-      <CopyableTextfield
-        v-if="selectedView === 'Config'"
-        :content="pairlistStore.configJSON"
-        :is-valid="pairlistStore.pairlistValid"
-      />
-      <CopyableTextfield v-if="selectedView === 'Results'" :content="pairlistStore.whitelist" />
+      <div class="position-relative flex-fill overflow-auto">
```

Comment:
This looks a lot better now

The only concern i have is on mobile - where the results view (nor the config) are visible at all.

i suspect it's caused by the reliance to provide "height" from the left hand panel - which isn't really working anymore once the layout flips to columns.